pom.xml

```diff
@@ -11,18 +11,20 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<jmh.version>1.11.2</jmh.version>
+		<jmh.version>1.22</jmh.version>
 		<uberjar.name>benchmarks</uberjar.name>
 
-		<pebble.version>2.2.0</pebble.version>
-		<mustache.version>0.9.1</mustache.version>
-		<freemarker.version>2.3.23</freemarker.version>
+		<pebble.version>2.4.0</pebble.version>
```

Comment:
Current pebble version is 3.1.2

```
<dependency>
	<groupId>io.pebbletemplates</groupId>
	<artifactId>pebble</artifactId>
	<version>3.1.2</version>
</dependency>
```

Reply:
done
thanks for the heads up 
upgraded also velocity-template which has a new artifactId